python/README.md

```diff
@@ -0,0 +1,81 @@
+Python Example
+==============
+
+The following with example CF deployment is an overview of the new AWS Elasticsearch service and highlights the differences between the legacy `elasticsearch56` and new `aws-elasticsearch` services available in the cloud.gov cloudfoundry marketplace.
+
+## Getting started
+
+The following example uses two services already created and available for consumption to your organization and space applications. The following services created for this example are named `old-es-service` (legacy elasticsearch56 service) and `new-es-service` (new aws-elasticsearch service).  Before running through this example, make sure to create the two services with the following cli commands:
+
+```bash
+##
+## Creating the following example services
+##
+
+### The legacy elasticsearch service
+$ cf create-service elasticsearch56 medium old-es-service
+
+### The new elasticsearch service
+$ cf create-service aws-elasticsearch BETA-es-dev new-es-service
+```
+
+To get started with this example, please clone or download this repository so you can follow along and run the steps from your computer.
+
+```bash
+## Clone and navitgate into example repository
+$ git clone https://github.com/cloud-gov/aws-elasticsearch-example && cd aws-elasticsearch-example
+```
+
+## Deploying
+
+The following deployment instructions assumes that you have cloned or downloaded this `aws-elasticsearch-example` and have navigated into the root of this directory (`$ cd python/`).  From the `python` directory push the CF deployment to you cloud.gov space.
+
+```bash
+$ cf push -f manifest.yml
+```
+
+The following apps are deployed into your space:
+- `elastic-tasks` is a CF app task runner used to load data and indices into the elasticsearch services.
```

Comment:
Same suggestions as above regarding this list:

```suggestion
When the `cf push` command finishes, the following apps are deployed into your cloud.gov space:
- `elastic-tasks` - an app task runner that is used to load data and indices into the Elasticsearch services.
```

Or, if this is the only app, perhaps this:

```suggestion
When the `cf push` command finishes an `elastic-tasks` app task runner will be deployed, which is used to load  data and indices into the Elasticsearch services.
```